src/SearchResults.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import fakeBookings from "./fakeBookings.js";
+
+import Tbody from "./Tbody.js";
+// console.log(fakeBookings);
+let SearchResults = fakeBookings => {
+  return (
+    <table className="table table-borderless">
+      <thead>
+        <tr>
+          <th scope="col">id</th>
+          <th scope="col">title</th>
+          <th scope="col">first name</th>
+          <th scope="col">surname</th>
+          <th scope="col">email</th>
+          <th scope="col">room id</th>
+          <th scope="col">check in date</th>
+          <th scope="col">check out date</th>
+          <th scope="col">days stayed</th>
+        </tr>
+      </thead>
+      <Tbody />
+    </table>
```

Comment:
This is totally ok. But, don't you think would make more sense to have a <BookingsTable /> component rather than a component with just the <Tbody />?

Reply:
i see mistake thread, may be fixing that will make my app better.

Reply:
i found this part very confusing especially splitting the table which i copied with bootstrap styles and break it in two parts was difficult. i was not  expecting some things to have perfect logic as long as it was working,i required help at this point

Reply:
probably having just <th> component would be cleaner, but at that moment required additional logic, i thought about exporting object values and keys into separate arrays and then mapping  into <th> and had no simpler way of doing that.

Reply:
Ok, let’s try to find some time this week and try to do it together. What you think?